.github/workflows/CI.yml

```diff
@@ -18,6 +18,18 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
         architecture: x64
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: 1.61
+    - name: Install system dependency
+      shell: bash
+      run: |
+        pip install dill
```

Comment:
Yes, because of the test below, when (de)serializing we import the `dill` library from the given python inside Rust, see `src/pyo3_extensions/mod.rs` lines 118 and 140, so `dill` is required for Rust tests to run